streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -514,8 +496,6 @@ public void shouldTransitToRestoringThenRunningAfterCreation() throws IOExceptio
         assertEquals(RUNNING, task.state());
         assertTrue(source1.initialized);
         assertTrue(source2.initialized);
-
-        EasyMock.verify(stateDirectory);
```

Comment:
I think some verifications are missing here, aren't they? For example, `stateManager.registerStore()`.

Reply:
Yeah, this is where I should have been more explicit:
```
EasyMock.expect(stateManager.changelogPartitions()).andReturn(singleton(changelogPartition));
```
is reported as an unnecessary stub and
```
stateManager.registerStore(stateStore, stateStore.stateRestoreCallback, null);
```
if verified is reported as not called